### ISSUE TITLE
Tyler optional task list.I may be able to get you a working holiday in Bali.But not promising.

### ISSUE THREAD

**Issue by Morningstar88** [Morningstar88/kalki-search#197]
**Task one, please sign up to GitHub.**2 minutes.No pressure, totally up to you.

**Comment by taiwanluthiers** [comment]
signed up. I hate website with their security BS making me use password that I will definitely forget in a minute.

**Comment by Morningstar88** [comment]
I'm sorry we had to put you through all of that.Are you feeling OK?Would you like to go lie down?Now we can talk in public

**Comment by Morningstar88** [comment]
I think step one is to ask Google to Gemini.Is Vitamin C or the Kintamani fly problem most interesting to you?I understand you like to do things one by one and that's really cool.

**Comment by taiwanluthiers** [comment]
I need to go lie down.

**Comment by Morningstar88** [comment]
OK, that's good.We are back on DEFCON 3 now.

**Comment by Morningstar88** [comment]
OK, we will focus on the fly problem.I am not making bleach.It's hypochlorous acid. Very different.And it's very, very cheap.if we're going to cooperate, we need a set of rules about how we think through this.Like a process--Can you open a Google doc? Please do that.

**Comment by taiwanluthiers** [comment]
https://docs.google.com/document/d/1WLAdagnGrs9EAgVRHzGX1DYS3hnjJ6rJ0oT2IN9EC0M/edit?usp=sharing

Here is a blank doc. Type what you need into it.

Bleach is sodium hypochlorite. I'm sure there's some hypochlorous acid inside. It's caustic as hell anyways. If you pass electricity through a salt solution it basically makes it, it causes a series of reaction that ends up making sodium chlorate when you let it go on at a high temperature (as in above about 30C)

**Comment by Morningstar88** [comment]
Thanks for putting that up.The hypochlorous acid comes from pool shock.Calcium hypochlorite.Not sodium.When you add it to water, it makes it very gentle bleach.I know what I'm talking about.I have actually done this.I am actually using it.
--
Ask yourself this question: 'Will this matter a year from now?'



Mar 27, 2024, 20:35 by ***@***.***:

**Comment by Morningstar88** [comment]
The hypochlorous acid is absolutely not caustic.You can even get it in your eyes. It is gentler than citric acid.I appreciate that you were close, but we need to get very scientific here.

--
Ask yourself this question: 'Will this matter a year from now?'



Mar 27, 2024, 20:35 by ***@***.***:

**Comment by Morningstar88** [comment]
Yes, it's very interesting stuff actually. It's the most prevalent acid in our immune systems.I use it in the bathroom and everything.It smells like a softer chlorine.It really improves my health.Basically.A pool shock footbath is quite amazing.

--
Ask yourself this question: 'Will this matter a year from now?'



Mar 27, 2024, 20:59 by ***@***.***:
